dynamite_flask_app/init_db.py

```diff
@@ -0,0 +1,15 @@
+"""This module creates all the neccessary objects that will be used in the application."""
```

Comment:
this creates all necessary objects then it should not be named as init_db.

Reply:
Named it init.py, do suggest a better name for it if you think this doesn't work.